release/release.md

```diff
@@ -0,0 +1,112 @@
+# How to do a Kata Containers Release
+
+Hi if you are reading this document you may also want to create a kata Containers Release.
+
+The Kata Containers Release Process is defined in the follwoing documents.
+
+https://github.com/kata-containers/documentation/blob/master/Releases.md
+
+To simply the process of read each release we have created a checklist for it.
+
+https://github.com/kata-containers/documentation/blob/master/Release-Checklist.md
+
+In order to simplify the process of do the Release Checklist we have semi-automated most of the process.
+
+So lets get started.
+
+## Requeriments 
```

Comment:
Yeah, it is safe to use in your system, most of the task are done in containers. 
- I'd love to add some tests for the steps, as last release we broken some of this scripts.
- I have an azure pipeline to create static tarballs from master @chavafg  I am not sure if is fine to add it, we depend all our jobs in jenkins, but was really easy to use.
- Added a note that the steps can be repeated in case something fails, not needed to start over.